robosuite/scripts/demo_device_ik_control.py

```diff
@@ -87,7 +87,8 @@
         env.render()
 
         # rotate the gripper so we can see it easily
-        env.set_robot_joint_positions([0, -1.18, 0.00, 2.18, 0.00, 0.57, 1.5708])
+        #env.set_robot_joint_positions([0, -1.18, 0.00, 2.18, 0.00, 0.57, 1.5708])
+        env.set_robot_joint_positions([0, np.pi / 16.0, 0.00, -np.pi / 2.0 - np.pi / 3.0, 0.00, np.pi - 0.2, -np.pi/4])
```

Comment:
This new initial pose doesn't work for Sawyer. Can you add a if-condition to use the previous pose for Sawyer and this new pose for Panda?

Reply:
done